app/views/main.scala.html

```diff
@@ -38,6 +38,15 @@
                     @Messages("main.header")
                 </a>
             </div>
+            @if(isLoggedIn) {
+                <ul class="govuk-header__navigation govuk-header__navigation--end">
+                    <li class="govuk-header__navigation-item">
+                        <a href="/logout" class="govuk-header__link">
```

Comment:
How does the /logout route work? Is it a default Pac4J thing?

I'm just wondering if we can think of some clearer routes than `/logout` and `/sign-out` because they're so similar that it's hard to work out what's going on.

Reply:
You can change them. `/logout` is a Pac4J think but it's defined in the routes file and you can change it. What should we change it to though?

Reply:
Maybe `/sign-out` as the Pac4J route, since that's the one that does the sign out action? Is the action in the SignOutController just used to render the page? If so, maybe call it `/signed-out`? I think you could work out the difference between `/sign-out` and `/signed-out`, though you could check with someone like Tom to see if they make sense to him out of context.

Reply:
They make sense to me. I've changed them

Reply:
Cheers. Probably makes sense to rename the view too, for consistency.

Reply:
Yep, make sense. I've updated the entries in the messages file as well.